vendors/nuvoton/boards/numaker_iot_m487_wifi/aws_demos/application_code/main.c

```diff
@@ -54,6 +62,7 @@
 #define mainLOGGING_TASK_STACK_SIZE         ( configMINIMAL_STACK_SIZE * 5 )
 #define mainLOGGING_MESSAGE_QUEUE_LENGTH    ( 25 )
 
+extern uint8_t ucMACAddress[ 6 ];
```

Comment:
Please use Macro instead of number for MAC length representation.

Reply:
@andysun2015  Fixed it in commit 5bf2f3f.